.github/workflows/rd_storage_visualization.yaml

```diff
@@ -0,0 +1,49 @@
+name: Storage visualization
+
+on:
+  # Run every Sunday morning.
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
can we make it like 4am on Monday. I  always end up looking at these on Sunday night when they come through and I really do not want to.